modules/arrow/src/lib/parse-arrow-sync.js

```diff
@@ -17,5 +17,28 @@ export default function parseArrowSync(arrayBuffer, options) {
     columnarTable[field.name] = values;
   });
 
-  return columnarTable;
+  switch (options.arrow.rowFormat) {
```

Comment:
Hmm I now realize that `rowFormat` may not be the best name for an option that chooses between columns and rows. But this will do for now until we rethink this.

Reply:
Probably something like `tableFormat: auto |  row-object | row-array | columnar | binary-columnar | arrow`